README.md

```diff
@@ -3,3 +3,44 @@
 <img src="https://media.giphy.com/media/143vPc6b08locw/giphy.gif" width="300">
 
 _Ship-it_ is Wattpad's tool for continuously deploying code.
+
+## Local Development
+
+This project uses skaffold for local development and testing. To get started,
+you'll need to install these tools locally:
+
+* helm: https://github.com/helm/helm
+* minikube: https://github.com/kubernetes/minikube
+* skaffold: https://github.com/GoogleContainerTools/skaffold
+
+1. Create a new minikube cluster
+
+```bash
+$ minikube create
```

Comment:
The command needed here is actually `minikube start`. Running this command also seemed to automatically set cluster context for `kubectl`.